Mathlib/Topology/MetricSpace/PiNat/Normed.lean

```diff
@@ -0,0 +1,60 @@
+/-
+Copyright (c) 2025 Yakov Pechersky. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yakov Pechersky
+-/
+import Mathlib.Analysis.Normed.Order.Basic
+import Mathlib.Topology.MetricSpace.PiNat.Basic
+
+/-!
+# Normed spaces `Π (n : ℕ), E n`
+
+When `E n` are topological spaces, the space `Π (n : ℕ), E n` is naturally a topological space
+(with the product topology). When `E n` are uniform spaces, it also inherits a uniform structure.
+However, it does not inherit a canonical metric space structure of the `E n`. Nevertheless, one
+can put a noncanonical metric space structure (or rather, several of them).
+This is handled in `Mathlib/Topology/MetricSpace/PiNat/Basic.lean`.
+Here, we extend the result to the normed additive group structure.
+
+## Main definitions and results
+
+* `PiCountable.normedAddCommGroup`
+
+-/
+
+noncomputable section
+
+namespace PiCountable
+
+variable {ι : Type*} [Encodable ι] {F : ι → Type*} [∀ i, NormedAddCommGroup (F i)]
+
+attribute [local instance] PiCountable.metricSpace
+
+@[simp]
+lemma dist_add_right (x y c : Π i, F i) :
```

Comment:
I think this should be a `have`, since it is immediately superseded by the instance below